src/game/g_lua.c

```diff
@@ -1349,17 +1348,8 @@ gentity_t *G_Lua_CreateEntity(char *params)
 
 		level.numSpawnVars++;
 	}
-	create = G_SpawnGEntityFromSpawnVars();
 
-	//create->classname = "lua_spawn"; // make additional param?
-
-	if (!create)  // don't link NULL ents
-	{
-		return NULL;
-	}
-
-	trap_LinkEntity(create);
-	return create;
+	return G_SpawnGEntityFromSpawnVars();
```

Comment:
For clarity: `G_SpawnGentityFromSpawnVars` returns `NULL` if the creation fails, so we don't need to explicitly check for that.